packages/optimistic-oracle/index.js

```diff
@@ -121,6 +121,8 @@ async function Poll(callback) {
       errorRetriesTimeout: process.env.ERROR_RETRIES_TIMEOUT ? Number(process.env.ERROR_RETRIES_TIMEOUT) : 10,
       // If there is an optimistic oracle config, add it. Else, set to null. Example config:
       // {
+      //   "disputePricePrecisionOfError":4 -> Proposal prices that do not equal the dispute price
```

Comment:
the default you have in the `proposer.js` is 6, not 4.
```suggestion
      //   "disputePricePrecisionOfError":6 -> Proposal prices that do not equal the dispute price
```